docs/architecture.md

```diff
@@ -2,65 +2,38 @@
+The project builder itself is based on the same project structure like the [starwit/reacthook-spring-template](https://github.com/starwit/reacthook-spring-template). The following description is valid for both. The project, based on Spring Boot is build via maven and is parted in subprojects shown in the diagram below.
 
-The following subprojects exist in every project built with lirejarp project builder:
-
-### persistence 
-This subproject contains classes for database access. Transactions are ensured through Enterprise Java Beans, if the database supports transactions.
+![Components projectBuilder](diagrams/projectbuilder-architecture-intern.drawio.svg)
 
+The following subprojects exist in every project built with projectBuilder:
+### application
+This subproject defines the over-all spring boot application. It contains spring boot application configuration and is needed to build the application.
+### persistence
+This subproject contains classes for database access. In this project, all database scripts are placed. Through flyway, these scripts are executed automatically during project start. Two databases has to be supported with the scripts: h2 (for tests) and mysql.
 ### rest
 This subproject enables the access from UI via RESTful webservices.
 ### webclient
-The UI logic is placed in the webclient project. ReactJS is used. The Frontend is utilized with [Material-UI](https://www.mui.com) and its design structure (mostly based on [JSS](https://cssinjs.org/?v=v10.8.1)).
-
-### packagewar
-The deployment of the project is done via war-file. It contains the projects persistence and rest as jar-files and the webclient. The project packagewar is resonsible for the build.
-
-![Component Diagram LireJarp](diagrams/lirejarpTemplate.png)
-
-## Additional projects in lirejarp project builder
-In lirejarp project builder, two additional projects are implemented:
+The UI logic is placed in the webclient project. React is used. The Frontend is utilized with [Material-UI](https://www.mui.com) and its design structure (mostly based on [JSS](https://cssinjs.org/?v=v10.8.1)).
 
+## ProjectBuilder specific
+In lirejarp ProjectBuilder, one additional project is implemented:
 ### generator
-The generator project contains all logic to generate projects configured with the lirejarp project builder.
-
-### dbautoinit
-In this project, all database scripts are placed. Through flyway, these scripts are executed automatically in the DEV-mode. Two databases are supported: h2 and mysql. In system.properties of the tomee-configuration, you can configure, which database is used: flyway.database=h2 lirejarp project puilder with the additional subprojects.
-
-![Component Diagram LireJarp Project Builder](https://wp.starwit.de/ljprojectbuilder/wp-content/uploads/sites/7/2017/02/komponentendiagrammPS.png)
-
-## About implementing data access
+The generator project contains all logic to generate projects configured with the ProjectBuilder.
```

Comment:
Please find one way to write ProjectBuilder. There are variations like projectbuilder,  project builder, etc